Main.cs

```diff
@@ -1276,11 +1282,9 @@ void PlayerInteract(Player pl, Message m)
 
 						long time_now = getMTime();
 						float ban_time = info.getInt(args[2]);
-						if (ban_time <= 0) {
-							ban_time = 0.5f;
-						} else if (ban_time > 48) {
-							ban_time = 48;
-						} else time_correct = true;
+						if (ban_time >= 0.5 || ban_time <= 72) {
+							time_correct = true;
```

Comment:
Player will get the "Banned" message even if `time_correct` stays `false`. Either throw a message about being kicked or do not kick.